vsphere/resource_vsphere_compute_cluster_test.go

```diff
@@ -982,6 +1010,60 @@ resource "vsphere_compute_cluster" "compute_cluster" {
 	)
 }
 
+func testAccResourceVSphereComputeClusterStretchedClusterEnabled() string {
+	return fmt.Sprintf(`
+%s
+
+resource "vsphere_compute_cluster" "compute_cluster" {
+  name                        = "testacc-compute-cluster"
+  datacenter_id               = data.vsphere_datacenter.rootdc1.id
+  host_system_ids             = [data.vsphere_host.roothost1.id, data.vsphere_host.roothost2.id]
```

Comment:
I'm unfamiliar with vSAN in general, but for the record the CI and expected testing environment is setup as follows:

`roothost1` Physical/main ESXi host expected to be in a default cluster we created on CI.
`roothost2` nested ESXi host expected to be in a default cluster we created on CI.
`roothost3` nested ESXi host you can feel free to move into a test specific cluster such as this one.
`roothost4` same as 3.

Probably you don't want to put roothost1 in this cluster for sure, we can DM offline to get a sense for what you need for these tests.

Reply:
Also host 2,3,4 are only on a private network, 1 is on both public and private. Doesn't seem to affect the other vSAN tests in the past but thought I should add that context.

Reply:
We also have the setup of mixed nested and physical ESXis, but the performance might be unstable, so I'll replace hosts with roothost2/3/4 in case, and let's have a try to see if anything else needed.

Reply:
To test stretched cluster, we could use roothost3 and roothost4 in primary and secondary fault domains, but we still need one more host outside of any cluster to be witness node, roothost1 and 2 doesn't meet this condition, so maybe this acceptance test will be failed as expected in CI, do we need keep code here and skip it? Or just let it failed and ignore its result.

Reply:
I suggest the failure should be expected.

Reply:
IMHO this vSAN case requires specific testing infrastructure which is not supported by the Acceptance environment. Looks we should consider skipping the test instead of accepting that it fails. What about  introducing some env variables like TF_VSPHERE_VSAN_HOST_1, TF_VSPHERE_VSAN_HOST_2 and TF_VSPHERE_VSAN_WITNESS_HOST which will be the minimal setup for the stretched cluster
and add some info in the skip message or in the function which checks if they are present and set to describe what are the requirements for those hosts ?